README.md

```diff
@@ -85,7 +85,6 @@ addStyles(dynamicCss, 'tag-id');
     font: "font(--my-font2)";                                               /* will use the overridden default unless it was defined in settings  */
     border-width: "unit(--var-from-settings, px)";                          /* will produce border-width: 42px */
     color: "fallback(color(--var-from-settings), color(color-8))";          /* will return the first none falsy value from left to right */
-    background-color: "smartContrast(color(--base-color), color(--contrast-color))"; /* given a base color and a suggested contrast color, returns the given contrast color if it's A11Y compliant or a lightened/darkened color that will comply */
```

Comment:
don't you want to document the new function?

Reply:
I think it's best to go rogue for now.
And once we start using it in wix-ui-tpa and see that it's working well, I'll add the documentation.
wdyt?

Reply:
ok